sessions/js-variables-and-numbers/social-media-post/js/index.js

```diff
@@ -0,0 +1,22 @@
+console.clear();
+
+/*
+1. Create the data for a little social media post. Create variables for the following data using the const keyword:
+	- a title
+	- a text
+	- the number of likes
+	- the user who created the post
+	- a boolean called isReported
+*/
+
+// --v-- write your code here --v--
+
+// --^-- write your code here --^--
+
+/*
+2. Log all variables to the console. Then increase the likes by one and log the amount of likes again. Adapt your code from 1.1 if necessary.
```

Comment:
Gramatically speaking, you wouldn't start a sentence with "then". You could just replace the full-stop with a comma.

Reply:
Good catch. I just removed the "Then"